RELEASE-PLANNING.md

```diff
@@ -1,18 +1,64 @@
 ## Release Planning 
 This section outlines the planned release schedule and key milestones for the FOCUS project. It includes the scope and high-level system requirements for each version, as well as detailed timelines for the development and review processes for each present and past releases. 
 
-### FOCUS Releases Scope
+### Scope of Upcoming Releases
+
+<table>
+    <tr>
+        <th>Version</th>
+        <th>Release By</th>
+        <th>Scope | High-Level System Requirements</th>
+    </tr>
+        <tr>
+        <td>v1.2</td>
+        <td>June 2025</td>
+        <td>
+            <strong>Initial Software as a Service (SaaS) Support</strong>
+            <ul>
+                <li>Addition of columns to support SaaS-centric concepts and billing models
+                </li>
+            </ul>
+            <strong>Continued expansion of Cloud Service Provider concepts</strong>
+            <ul>
+                <li>Addition of Invoice ID to support invoice reconciliation use cases
+                </li>
+                <li>Addition of SKU properties that are prominent / common across providers
+                </li>
+            </ul>
+            <strong>Fixes and clarifications</strong>
+            <ul>
+                <li>Revisions of existing specification content to increase consistency and reduce ambiguity</li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td>v1.3</td>
+        <td>Late 2025</td>
+        <td>
+            TBD
+        </td>
+    </tr>
+</table>
+
+### Scope of Previous Releases
 <table>
     <tr>
         <th>Version</th>
         <th>Release By</th>
         <th>Scope | High-Level System Requirements</th>
+    </tr>
+        <tr>
+        <td>v1.0</td>
+        <td>June 2024</td>
+        <td>
+            TBD
```

Comment:
@jpradocueva @udam-f2 Do we have a short write-up of 1.0 themes in a previously-authored artifact that we can add here?  I looked through previous versions of this doc, and I don't believe that was ever added here.  Obviously all the details are in the [changelog](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/CHANGELOG.md), but it would be nice to have a high-level summary for all releases in a single place.

Reply:
My super-picky feedback is that "Addition of" is redundant, but ignore that.  I'm good with this.  thanks for doing this.